ci/nodepool/nodepool.yaml

```diff
@@ -1,12 +1,12 @@
-script-dir: /home/pulp/pulp-ci/ci/nodepool/scripts/
+script-dir: /home/pulp/pulp_packaging/ci/nodepool/scripts/
 
 # empty dir, stops nodepool from throwing a traceback on start
 images-dir: /etc/nodepool/images
 
 dburi: 'postgresql:///nodepool'
 
 zmq-publishers:
-  - tcp://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com:8881
+  - tcp://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com:8888
```

Comment:
Has Jenkins' ZeroMQ configuration changed?

Reply:
I am unsure of when the Jenkins configuration change actually occured. I grabbed this from the current config version on nodepool. 
I'll push an update separating out the commit to update this config to what's currently in nodepool, and the f27 changes I'm proposing.